files/tasks/rpm-test-deps.yaml

```diff
@@ -2,7 +2,7 @@
 - name: Install test rpm dependencies
   dnf:
     name:
-      - python3-flexmock
+      #      - python3-flexmock
```

Comment:
Wouldn't this work?

```suggestion
      - python3-flexmock-0.10.4
```

Reply:
it would, for now, until that build is "garbage collected" from fedora repos

Reply:
but you're right, we can do it your, easier way, since it's just a workaround and figure out something better once the build disappears